docs/radio.rst

```diff
@@ -0,0 +1,67 @@
+Radio
+*****
+
+.. py:module:: radio
+
+The ``radio`` module allows devices to work together via simple wireless
+networks.
+
+Functions
+=========
+
+.. py:function:: send(message)
+
+    Sends a message, expressed as a string.
+
+.. py:function:: receive()
+
+    Receive the next incoming message on the message queue. Returns ``None`` if
+    there are no pending messages.
+
+.. py:function:: config(length=32, queue=3, channel=7, power=0, address=b'uBit\x00', data_rate=radio.MODE_1MHZ)
+
+    Configures the various settings relating to the radio. The specified
+    default values are sensible.
+
+    The ``length`` defines the size, in bytes, of a message sent via the radio.
+    It can be up to 256 (????? CHECK THIS from memory) bytes long.
+
+    The ``queue`` specifies the number of messages that can be stored on the
+    incoming message queue.
```

Comment:
Is this a circular queue? We should probably indicate what happens when receiving more messages than there are spaces left.